lib/Test2/Harness/Collector.pm

```diff
@@ -271,6 +308,12 @@ sub jobs {
         );
     }
 
+    # The collector didn't read in all the jobs because it'd run out of file handles. We need to let the harness output know we're behind.
+    if( $max_open_jobs <= scalar keys %$jobs ) {
+        my $msg = "The Yath Collector is running behind. More than $max_open_jobs test results have not been processed.";
+        $self->send_backed_up or $self->{+ACTION}->( $self->_harness_event(0, undef, time, info => [{details => $msg, tag => "INTERNAL", debug => 1, important => 1}]) );
```

Comment:
that message should not be displayed once

Reply:
Right. That's what is happening.